ansys/dpf/core/plotter.py

```diff
@@ -74,6 +74,37 @@ def add_field(self, field, meshed_region=None, **kwargs):
         # plot
         self._plotter.add_mesh(meshed_region.grid, scalars=overall_data, **kwargs)
 
+        if show_max or show_min:
+            # Get Min-Max for the field
+            min_max = core.operators.min_max.min_max()
+            min_max.inputs.connect(field)
+
+        # Add Min and Max Labels
+        _labels = []
+        _grid_points = []
+        if show_max:
+            max_value = min_max.outputs.field_max().data[0]
+            # Get max value index
+            max_value_index = np.where(overall_data == max_value)[0][0]
+            # Get node ID at max value
+            _node_id_at_max = min_max.outputs.field_max().scoping.ids[0]
+            _labels.append("Max: %s, NodeID: %s" % (round(max_value, 2), _node_id_at_max))
+            _grid_points.append(meshed_region.grid.points[max_value_index])
```

Comment:
Remove leading underscore.
```suggestion
            node_id_at_max = min_max.outputs.field_max().scoping.ids[0]
            labels.append("Max: %s, NodeID: %s" % (round(max_value, 2), node_id_at_max))
            grid_points.append(meshed_region.grid.points[max_value_index])
```